containers/cpp-mariadb/.devcontainer/Dockerfile

```diff
@@ -7,23 +7,5 @@ FROM mcr.microsoft.com/vscode/devcontainers/cpp:0-${VARIANT}
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install curl
 
-# RUN curl https://downloads.mariadb.com/MariaDB/mariadb_repo_setup \
-#     && echo "5feb2aac767c512cc9e2af674d1aef42df0b775ba2968fffa8700eb42702bd44  mariadb_es_repo_setup" | sha256sum -c - \
-#     && chmod +x mariadb_repo_setup \
-#     && ./mariadb_repo_setup --mariadb-server-version="mariadb-10.6" \
-#     && apt install -y libmariadb3 libmariadb-dev \
-#     && curl https://dlm.mariadb.com/1683461/connectors/cpp/connector-cpp-1.0.1/mariadb-connector-cpp-1.0.1-debian-buster-amd64.tar.gz \
-#     && tar -xvzf mariadb-connector-cpp-*.tar.gz \
-#     && cd mariadb-connector-cpp-*/ \
-#     && install -d /usr/include/mariadb/conncpp \
-#     && install -d /usr/include/mariadb/conncpp/compat \
-#     && cp -R ./include/mariadb/* /usr/include/mariadb/ \
-#     && cp -R ./include/mariadb/conncpp/* /usr/include/mariadb/conncpp \
-#     && cp -R include/mariadb/conncpp/compat/* /usr/include/mariadb/conncpp/compat \
-#     && install -d /usr/lib/mariadb \
-#     && install -d /usr/lib/mariadb/plugin \
-#     && cp lib/mariadb/libmariadbcpp.so /usr/lib \
-#     && cp -R lib/mariadb/plugin/* /usr/lib/mariadb/plugin
-
 COPY ./install-mariadb.sh /
 RUN chmod +x /install-mariadb.sh && ./install-mariadb.sh
```

Comment:
If you create the file with +x, no need to `chmod` it as part of the `RUN` call

Reply:
Docker best practices seems to be to not make the file `+x` on creation but on run.